recipes/pcaspy/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "0.7.2" %}
+
+package:
+  name: pcaspy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pcaspy/pcaspy-{{ version }}.tar.gz
+  sha256: 823e5b3c25dcb73795730aee4f656e21fb4f590f866fcf6adb2d56cc6b5ced3d
+  patches:
+    - readline.patch  # [win]
```

Comment:
Could you please write some explanation why this patch is needed?

Reply:
@marcelotrevisani, see 42bc982